shared/utils/combine-remediations.py

```diff
@@ -423,21 +423,21 @@ def main():
                         else:
                             mod_file += line
 
-                complexity = None
-                disruption = None
-                reboot = None
+                complexity = "unknown"
+                disruption = "unknown"
+                reboot = "false"
```

Comment:
is this a safe default? saying that fix doesn't require reboot when we are not sure doesn't seem right.